skt/runner.py

```diff
@@ -135,7 +136,7 @@ def dumpjunitresults(self, jobid, junit):
         with open(fname, 'w') as fileh:
             bkr = subprocess.Popen(args, stdout=fileh)
             bkr.communicate()
-        if bkr.returncode != 0:
+        if bkr.returncode != SKT_SUCCESS:
```

Comment:
This one is a subprocess return code which shouldn't be compared with skt results.